src/commands/clean.stanza

```diff
@@ -9,7 +9,7 @@ defpackage slm/commands/clean:
 
 public defn clean (cmd-args:CommandArgs) -> False:
   debug("cleaning .slm")
-  for build-dir in [SLM_DEPS_DIR, SLM_PKGS_DIR] do:
+  for build-dir in SLM_BUILD_DIRS do:
```

Comment:
Presumably this will fail if there is not a cache directory specified? Seems like that doesn't have to be a breaking change -- particularly since caching is on by default. 
Will it also fail if the user manually deletes one of these directories before calling `clean`?

Reply:
Yes - you are correct. That is a bug. I will make this more forgiving if the `pkg-cache` directory is turned off.

Reply:
Addressed. Thank you!